lib/kafo/data_type_parser.rb

```diff
@@ -13,7 +13,7 @@ def initialize(manifest)
       type_line_without_newlines = +''
       manifest.each_line do |line|
         line = line.force_encoding("UTF-8").strip
-        next if line.start_with?('#')
+        next if line.start_with?('#') || line.empty?
```

Comment:
This won't catch a line full of whitespace (which some editors might leave as is).
```suggestion
        next if line.start_with?('#')
        next if line.strip.empty?
```

WDYT?

Reply:
oh wait, we already strip above, NVM